core/modules/entity/tests/entity_query.test

```diff
@@ -8,6 +8,21 @@
  * Tests EntityFieldQuery.
  */
 class EntityFieldQueryTestCase extends BackdropWebTestCase {
+  /**
+   * @var array
+   */
+  protected $fields;
+
+  /**
+   * @var array
+   */
+  protected $field_names;
+
+  /**
+   * @var array
+   */
+  protected $instances;
```

Comment:
Another instances of a property that may not be needed after all. Only used once in the whole test, during setUp. It's unclear why.

Reply:
It's not needed, not used anywhere after creation, neither is the class extended, so it can go. :+1: 

To clarify: I'm fine with adapting tests, as long as the change is trivial (as here). Also because trivial changes don't get too hard to review. :wink: